criu/cr-restore.c

```diff
@@ -3360,16 +3354,31 @@ static bool groups_match(gid_t *groups, int n_groups)
 	return ret;
 }
 
+static int copy_caps(u32 *out_caps, u32 *in_caps, int n_words)
+{
+	int last_cap = min(kdat.last_cap, 32 * CR_CAP_SIZE - 1);
```

Comment:
sounds like a bug. You need to add a comment to explain when it can happen.

Reply:
Added a check to kerndat instead.